core/src/main/java/org/mule/runtime/core/internal/processor/chain/SubflowMessageProcessorChainBuilder.java

```diff
@@ -100,7 +101,8 @@ private static class SubFlowMessageProcessorChain extends DefaultMessageProcesso
     }
 
     private void pushSubFlowFlowStackElement(CoreEvent event) {
-      ((DefaultFlowCallStack) event.getFlowCallStack()).push(new FlowStackElement(subFlowName, null));
+      ((DefaultFlowCallStack) event.getFlowCallStack())
+          .push(new FlowStackElement(subFlowName, null, buildFromStringRepresentation("subflow")));
```

Comment:
use a constant for `buildFromStringRepresentation("subflow")`, to avoid building the location on every processed event